generators/language-swift-kitura/bluemix-label-mappings.json

```diff
@@ -2,7 +2,6 @@
   "cloudant": "cloudant",
   "objectStorage": "object_storage",
   "appid": "appid",
-  "conversation": "watson_assistant",
```

Comment:
why is this mapping removed?

Reply:
The service was added as `watson_assistant` and therefore generating two sets of credentials. While the service name changed, the bluemix label didn't so we should keep using `conversation`.
In this PR i'm moving the templates to the conversation service. If you look at the state of `develop` you will find swift and go in `watson-assistant` and Node Python and Java in `conversation`.

I will be updating the other languages in the next PR.

Reply:
I am not sure about the implications on this for Go,  I think the changes here can proceed.

Reply:
I think the problem is `Swift credential reading library and generator-swiftserver both pass `assistant` inputs`.

I will rollback those changes but please don't take any more decision that involves the watson user experience without consulting with the right team. The fact that we now have `watson_assistant` and `watson_conversation` is a HORRIBLE experience.


I'm just going to limit my contributions to update the existing node, python and java template.